turbo.json

```diff
@@ -4,7 +4,19 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build"],
-      "env": ["*", "RESEND_API_KEY", "!NEXT_PUBLIC_GIT_*"],
+      "env": [
+        "*",
+        "RESEND_API_KEY",
+        "!NEXT_PUBLIC_GIT_*",
+        "DATABASE_URL",
+        "TINY_BIRD_API_KEY",
+        "CLERK_SECRET_KEY",
+        "NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY",
+        "NEXT_PUBLIC_CLERK_SIGN_IN_URL",
+        "NEXT_PUBLIC_CLERK_SIGN_UP_URL",
+        "NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL",
+        "NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL"
```

Comment:
Did you get an error on building and it solved the issue? Asking because I thought `*` would be enough and I got an error with the `RESEND_API_KEY`.